packages/global/components/blocks/marko.json

```diff
@@ -45,6 +45,7 @@
     "@type": "string",
     "@name": "string",
     "@aliases": "array",
+    "@primary-section-alias": "string",
```

Comment:
This should be 
```
"@display-ads": {
  "type": "boolean",
  "default-value": true,
}
```

This would ensure that the default functionality doesn't change.